interfaces/vrep/robots/LBR4pVrepRobot.m

```diff
@@ -0,0 +1,69 @@
+classdef LBR4pVrepRobot < DQ_VrepRobot
+    %UNTITLED Summary of this class goes here
+    %   Detailed explanation goes here
+    
+    properties
+        joint_names;
+        base_frame_name;
+    end
+    
+    methods
+        function obj = LBR4pVrepRobot(robot_name,vrep_interface)
+            %UNTITLED Construct an instance of this class
+            %   Detailed explanation goes here
+            obj.robot_name = robot_name;
+            obj.vrep_interface = vrep_interface;
+            
+            %This might be useful to every subclass
```

Comment:
This part deserves a little more explanation, because it indicates what one should do when implementing new classes.